### PR TITLE
Add Tencent WeChat

### PR DESCRIPTION
**Pull request by yuuow**
Add Tencent WeChat.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/369)

### CODE REVIEW

File: manifests/Tencent/WeChat/2.9.0.1000.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Id: Tencent.WeChat
+Name: Tencent WeChat
+Version: 2.9.0.1000
+Publisher: Xunco
+Author: Tencent Corporation
+Homepage: https://www.wechat.com
+Description: Connecting a billion people with calls, chats and more.
+License: Copyright (C) Tencent Corporation
+LicenseUrl: https://www.wechat.com/en/service_terms.html
+Tags: WeChat, WeXin, IM
```
Comment (ousugo) [inline manifests/Tencent/WeChat/2.9.0.1000.yaml:10]: "WeXin" might be "WeiXin"?

Comment (MeXunco) [reply manifests/Tencent/WeChat/2.9.0.1000.yaml:10]: Yes

**Comment by KevinLaMS** [comment]
I cannot get the app to install. It always fails with error "Installer failed with exit code: 3221226525".  
Are you familiar with what might be causing this?

**Comment by MeXunco** [comment]
> I cannot get the app to install. It always fails with error "Installer failed with exit code: 3221226525".
> Are you familiar with what might be causing this?

May I ask your system version? Is it win10?
Did you see this interface？
<details>
<summary>Click to view</summary>

installing

</details>

**Comment by MeXunco** [comment]
I can't find relevant issues through the search engine. I'm not sure if it's your environment.

**Comment by imba-tjd** [comment]
I can confirm that `-s` will cause the installation fails.

```
Faulting application name: WeChatSetup.exe, version: 2.9.0.1000, time stamp: 0x55c15ce6
Faulting module name: WeChatInstallDll.dll, version: 1.0.0.0, time stamp: 0x5e708e34
Exception code: 0xc000041d
Fault offset: 0x00019930
Faulting process id: 0x3dec
Faulting application start time: 0x01d62e6a0a5f5460
Faulting application path: E:\下载\WeChatSetup.exe
Faulting module path: C:\Users\xxx\AppData\Local\Temp\nsp73DE.tmp\WeChatInstallDll.dll
Report Id: a6f8d554-fc84-455d-950b-dabe037cdc34
Faulting package full name: 
Faulting package-relative application ID: 
```

**Comment by KevinLaMS** [comment]
I will try it again. I am using a  Windows 10 VM.  1909, 18363.  No, setup did not succeed, so I did not see the welcome page.

File: manifests/Tencent/WeChat/2.9.0.1000.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Tags: WeChat, WeiXin, IM
+MinOSVersion: 10.0.0.0
+InstallerType: exe
+Installers:
+  - Arch: x86
+    Url: https://dldir1.qq.com/weixin/Windows/WeChatSetup.exe
+    Sha256: 973b429c4f9c1882340c3b8bfa066ae159b5dd409f876538e3820efbeb0a5ab4
+    Switches:
+      Silent: -s
+      SilentWithProgress: -s
```
Comment (imba-tjd) [inline manifests/Tencent/WeChat/2.9.0.1000.yaml:19]:
```suggestion
      Silent: /S
      SilentWithProgress: /S
```

Comment (imba-tjd) [reply manifests/Tencent/WeChat/2.9.0.1000.yaml:19]: From https://github.com/facorread/wechat-chocolatey/blob/master/tools/chocolateyinstall.ps1

**Comment by KevinLaMS** [comment]
Ah.  That must be it.  I saw it fail with the client.   But then I ran just the URL, then it worked.
It looks the command is /S
https://www.manageengine.com/products/desktop-central/software-installation/silent_install_WeChat-2.4.5.73.html

File: manifests/Tencent/WeChat/2.9.0.1000.yaml (hunk 2 above)
Comment (MeXunco) [reply manifests/Tencent/WeChat/2.9.0.1000.yaml]: Thank you very much

Comment (imba-tjd) [reply manifests/Tencent/WeChat/2.9.0.1000.yaml]: emmmmm你这样做，我就不在作者列表里了。不过算了吧。

Comment (MeXunco) [reply manifests/Tencent/WeChat/2.9.0.1000.yaml]: 抱歉 ＞﹏＜，手速太快，非常感谢

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**